PP_APP/server/frontend/templates/post_view.html

```diff
@@ -0,0 +1,59 @@
+{% extends "base.html" %}
+{% load static %}
+{% block content %}
+
+<section id="about" class="about-section bg-light">
+    <div class="container">
+        {% for key in pp_post %}
+            <h1>POST #{{key.img_post.id}}</h1>
+            <table class="table">
+                <tr>
+                    <th>User</th>
+                    <td>{{key.img_post.user_id.email}}</td>
+                </tr>
+                <tr>
+                    <th>Top</th>
+                    <td><a href="../../media/{{key.img_post.top_img}}" target="blank">{{key.img_post.top_img}}</a></td>
+                </tr>
+                <tr>
+                    <th>Side</th>
+                    <td><a href="../../media/{{key.img_post.side_img}}" target="blank">{{key.img_post.side_img}}</a></td>
+                </tr>
+                <tr>
+                    <th>Infer</th>
+                    <td><a href="../../media/{{key.img_post.infer_img}}" target="blank">{{key.img_post.infer_img}}</a></td>
+                </tr>
+                <tr>
+                    <th>Date</th>
+                    <td>{{key.img_post.date_posted}}</td>
+                </tr>
+                <tr>
+                    <th>Brand Name</th>
+                    <td>{{key.packager.brand_name}}</td>
+                </tr>
+                <tr>
+                    <th>Materials</th>
```

Comment:
We will also parse this after the beta to show a response in English instead of a JSON response. I can create a Github Issue for this to keep track of it.